core/scss/components/checkbox/module.scss

```diff
@@ -1,19 +1,45 @@
-@import '../../tools/functions/core';
+@import '../../tools/mixins/core';
 
-@import '../../colors/settings';
-@import '../../settings/core';
-@import '../../settings/media';
-@import '../../typography/settings';
-@import '../forms/settings';
-@import '../../settings/focus';
-@import 'settings';
+// Icons
 
-@import '../input/mixins';
+@mixin icon-clear_16() { content: char-from-code("f34a"); }
+@mixin icon-warning_16() { content: char-from-code("fa63"); }
+@mixin icon-check-circle_16() { content: char-from-code("f33d"); }
+@mixin icon-clear_12() { content: char-from-code("f348"); }
+@mixin icon-warning_12() { content: char-from-code("fa61"); }
+@mixin icon-check-circle_12() { content: char-from-code("f33b"); }
+@mixin icon-minus_16() { content: char-from-code("f65e"); }
+@mixin icon-minus_12() { content: char-from-code("f65c"); }
+@mixin icon-check_12() { content: char-from-code("f32d"); }
+
+// Settings
+@import '../../settings/settings';
+
+// Tools
 @import '../../tools/mixins/border-radius';
 @import '../../tools/mixins/flex';
 @import '../../tools/mixins/focus';
 @import '../../tools/mixins/spacing';
-@import '~@momentum-ui/icons/scss/mixins';
+// @import '../../typography/mixins';
+@mixin brand-font-face-regular {
+    @include exports('brand-font-face-regular') {
+      @font-face {
+        font-family: 'CiscoSansTT Regular';
+        font-style: normal;
+        font-weight: normal;
+        src: url('#{$brand-font-folder}/CiscoSansTTRegular.woff2') format('woff2'),
+          url('#{$brand-font-folder}/CiscoSansTTRegular.woff') format('woff');
+        font-display: fallback;
+      }
+    }
+  }
+
+  @import '../../components/input/mixins';
+@import '../../components/loader/mixins';
+
+// Component Helpers
+@import '../../components/input/input';
```

Comment:
Input is a lot of CSS to import. Let's figure out what is needed and perhaps create mixins to import just the needed CSS.

Reply:
I got it whittled down